content/workspace/openbb-copilot.md

```diff
@@ -12,336 +12,616 @@ keywords:
 
-<div style={{display: 'flex', justifyContent: 'center'}}>
-<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/ada_trash_can_icon.png" alt="clear chat button openbb copilot" width="100%" height="100%" />
+This feature ensures consistency across team members and dramatically reduces the time spent retyping similar requests.
+
+<div
+  style={{ display: "flex", justifyContent: "center", alignItems: "center" }}
+>
+  <img
+    src="https://openbb-cms.directus.app/assets/f145bb2d-89bf-49f9-8fd7-af24d78e4dff.png"
+    width="100%"
+  />
 </div>
 
-## General question answering
+### Model
+
+The OpenBB Copilot leverages cutting-edge large language models from OpenAI, specifically optimized for financial analysis and data interpretation tasks.
 
-The simplest functionality of OpenBB Copilot is its general question answering
-capability. For example, you can ask Copilot things like:
+For enterprise deployments, OpenBB provides seamless integration with your organization's Azure OpenAI or dedicated OpenAI deployments, ensuring data sovereignty, compliance with internal security policies, and the ability to customize model behavior according to your firm's specific requirements and risk parameters.
 
-- "What's the difference between stocks and bonds?"
-- "Explain inflation."
-- "Can you explain the concept of dollar-cost averaging?"
+If you would like full control over your agent capabilities, here's [our open source repository](https://github.com/OpenBB-finance/agents-for-openbb) with examples of AI custom agents that you can build and integrate into the OpenBB Workspace.
 
-For general questions, OpenBB Copilot will rely on its underlying model's
-extensive training data to formulate answers. We suggest you spend some time    
-exploring this functionality with a range of questions across different topics
-to get a feel for how OpenBB Copilot works and behaves.
+## Context
 
-## Interacting with data sources using OpenBB Copilot
+The Copilot employs a sophisticated context understanding system that intelligently prioritizes and processes multiple information sources simultaneously.
 
-For OpenBB Copilot, widgets are data sources that can be used to retrieve data relevant to answer your queries. OpenBB Copilot uses the current values of widget parameters (e.g. symbol, start date, etc.) as suggestions to answer your query, but it can also change them if required.
+This hierarchical approach ensures that the most relevant and specific data takes precedence when generating responses, while still maintaining awareness of broader workspace context and conversation history.
 
-This allows you to treat a widget as a data that can return data for multiple tickers and inputs, rather than just the data that is currently being displayed.
+### Priority
 
-Widgets have the following priority:
+The agent prioritizes context in the following order to provide the most relevant answers:
```

Comment:
Do we need the second part of the sentence here? Everything that comes after "order"